fs/osfs.py

```diff
@@ -420,17 +420,18 @@ def _check_copy(self, src_path, dst_path, overwrite=False):
 
     if sys.version_info[:2] < (3, 8) and sendfile is not None:
 
-        _sendfile_error_codes = frozenset(
-            {
-                errno.EIO,
-                errno.EINVAL,
-                errno.ENOSYS,
-                errno.ENOTSUP,  # type: ignore
-                errno.EBADF,
-                errno.ENOTSOCK,
-                errno.EOPNOTSUPP,
-            }
-        )
+        _sendfile_error_codes = {
+            errno.EIO,
+            errno.EINVAL,
+            errno.ENOSYS,
+            errno.EBADF,
+            errno.ENOTSOCK,
+            errno.EOPNOTSUPP,
+        }  # type: MutableSet[int]
```

Comment:
I think the type is redundant here. Mypy can deduce the types of literals.

Reply:
Does that mean the change on line 63 is also redundant?

Reply:
Probably